CHANGELOG.md

```diff
@@ -158,6 +158,7 @@ our [guidelines for writing a good changelog entry](https://github.com/biomejs/b
 #### Bug fixes
 
 - Fix [#4121](https://github.com/biomejs/biome/issues/4121). Respect line width when printing multiline strings. Contributed by @ah-yu
```

Comment:
In this line, it's not clear that the fix is for the CSS files

Reply:
should i update this here？ 😂，it seems not my PR CHANGELOG，Then I changed it together by the way?

Reply:
Oh, my bad! I didn't notice it wasn't you. It's fine then :)